CoreGraphFramework/release/modules/ext/docs/CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/constellation-file-format.md

```diff
@@ -324,40 +324,21 @@ on open.
 
 Although the "meta" section has the same "attrs"+"data" format as the
 other sections, the attributes are defined by modules, rather than being
-built-in types. For instance, the module that defines which attribute
-values are displayed as labels on icons saves its state in the attribute
-"labels" of type "labels". When the document is opened, the
-Constellation graph opener will find a "labels" attribute and advertise
-it to the current modules. The module responsible for labels will
-recognise the attribute, claim it, and read its state.
+built-in types. For instance, the module that defines what filters and 
+configurations to display in the Conversation View saves its state in the 
+attribute "conversation_view_state" of type "conversation_view_state". 
```

Comment:
When viewing in help the conversation_view_state text has removed the underscores and put the word view in italics